app/models/admin-event.server.ts

```diff
@@ -127,6 +129,27 @@ export const modelAdminEvent = {
       locationId = location.id
     }
 
+    if (event?.image && imageUrl) {
+      imageId = event.image.id
+      await prisma.eventImage.update({
+        where: {
+          id: event.image.id,
+        },
+        data: {
+          url: imageUrl,
+          altText: `${title} cover image`,
```

Comment:
@fikrialwan In image element, the alt doesn't need "image" text. So it would just be:

```ts
url: imageUrl,
altText: title
```

Reply:
I've remove the used of "image" text on alt on this commit https://github.com/bandungdevcom/bandungdev.com/pull/42/commits/c3c36b907d2fb15d956ab2751d45119694f47344